edgecontext/req_context.go

```diff
@@ -65,6 +65,13 @@ func (e *EdgeRequestContext) User() User {
 	}
 }
 
+// OriginService returns the info about the origin of this request.
+func (e *EdgeRequestContext) OriginService() OriginService {
```

Comment:
🔕 Should this just return a string instead, like [`SessionID`](https://pkg.go.dev/github.com/reddit/baseplate.go/edgecontext?tab=doc#EdgeRequestContext.SessionID)? Or do we expect that we would add more fields into origin service in the future?

Reply:
Hmm, I wouldn't say we currently have anything _planned_ and there's a good chance we wouldn't add any more fields but it seems like it would be a _little_ more likely than DeviceID/SessionID.